src/applications/check-in/pre-check-in/tests/e2e/app-reload/reload.page.error.cypress.spec.js

```diff
@@ -82,7 +82,7 @@ describe('Pre-Check In Experience', () => {
       NextOfKin.attemptToGoToNextPage();
       Confirmation.validatePageLoaded();
       cy.reload();
-      Error.validatePageLoaded();
+      Error.validatePageLoaded(true, true);
```

Comment:
I do think that we need to update this page class to either remove these vars or get them sorted out. Probably another ticket, but should get fixed up.

Reply:
I created an issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/50614